pkg/lang/frontend/starlark/runtime/runtime.go

```diff
@@ -90,3 +92,30 @@ func ruleFuncDaemon(thread *starlark.Thread, _ *starlark.Builtin,
 	}
 	return starlark.None, nil
 }
+
+func ruleFuncExpose(thread *starlark.Thread, _ *starlark.Builtin,
+	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var (
+		envdPort    starlark.Int
+		hostPort    = starlark.MakeInt(0)
+		serviceName = starlark.String("")
+	)
+
+	if err := starlark.UnpackArgs(ruleExpose,
+		args, kwargs, "envd_port", &envdPort, "host_port?", &hostPort, "service?", &serviceName); err != nil {
+		return nil, err
+	}
+	envdPortInt, ok := envdPort.Int64()
+	if !ok && envdPortInt < 0 && envdPortInt > 65536 {
+		return nil, errors.New("envd_port must be a positive integer less than 65536")
+	}
+	hostPortInt, ok := hostPort.Int64()
+	if !ok && hostPortInt < 0 && hostPortInt > 65536 {
```

Comment:
`65536` can not be used to bind, `0` also may be reserved.

Reply:
Invited you as the co-author of this commit.